sys/src/9/amd64/devacpi.c

```diff
@@ -1910,12 +1936,14 @@ static void acpiinitonce(void)
 	if (root != nil)
 		print("ACPI initialized\n");
 	addarchfile("acpimem", 0444, acpimemread, nil);
+	/* XXX these permissions are too loose? */
+	addarchfile("acpiio", 0666, acpiioread, acpiiowrite);
+	addarchfile("acpiintr", 0666, acpiintrread, acpiintrwrite);
```

Comment:
Mark this as exclusion open, i.e. single open at a time. This will make lots of other stuff easier.